pythran/pythonic/include/types/numpy_expr.hpp

```diff
@@ -428,6 +428,9 @@ namespace pythonic
 
       numpy_expr(Args const &... args);
 
+      numpy_expr& operator=(numpy_expr const&) = default;
+      numpy_expr& operator=(numpy_expr&&) = default;
```

Comment:
I just need the move operator. I changed them for an optional type so I think we can remove them.